main.js

```diff
@@ -28,6 +30,7 @@ module.exports = function(options) {
 		withAnonMiddleware: false,
 		withBackendAuthentication: false,
 		withRequestTracing: false,
+		hasHeadCss: false,
```

Comment:
Correct, though good point, might be something we need in the future. Though if an app has wildly different heads between pages, maybe it shouldn't be within a single app...